docs/settings.rst

```diff
@@ -26,9 +26,30 @@ Deprecated. Use ``SCRAPY_POET_RULES`` instead.
 SCRAPY_POET_RULES
 -----------------
 
-Default: ``None``
+Default: :meth:`web_poet.default_registry.get_rules()`
+
+.. warning::
+
+    Although ``SCRAPY_POET_RULES`` already has values from the ``default_registry``,
+    make sure you call :func:`web_poet.rules.consume_modules` if you're using
+    other rules from other external packages.
+
+    Example:
+
+    .. code-block:: python
+
+        from web_poet import default_registry, consume_modules
+
+        # The consume_modules() must be called first if you need to properly import
+        # rules from other packages. Otherwise, it can be omitted.
+        # More info about this caveat on web-poet docs.
+        consume_modules("external_package_A", "another_ext_package.lib")
+
+        # To get all of the Override Rules that were declared via annotations.
+        SCRAPY_POET_RULES = default_registry.get_rules()
```

Comment:
Isn’t the point that this line is not necessary given the default value?

Reply:
It's still necessary if rules from external packages are needed due to the `consume_modules()` needing to be called.

Otherwise, the default rules inside `SCRAPY_POET_RULES` is incomplete.

Reply:
But at which point is `get_rules` as a default value called? If it is called after settings have been defined, then defining `consume_modules` in `settings.py` should still work, right? If the current implementation calls get_rules too early, maybe we could change that. But I would check first, because I am not sure the current code needs to be changed.

Reply:
Aha, I see your point. The `default_registry.get_rules()` is only called after the settings has been loaded, so yes, we can avoid redefining `SCRAPY_POET_RULES` here.

Updated in c42618e.